src/system/api/utils/wordpress/rest.js

```diff
@@ -224,25 +224,31 @@ const posts = () => {
 		},
 
 		/**
-		 * get post with raw data by ID
-		 * @param id
-		 * @param config
-		 * @returns {Promise<*>}
+		 * Override WP post data with library post item data
 		 */
-		formedPost( id, config = {} ) {
-			config.cacheKey = 'posts'
-			return http.get( `fl-assistant/v1/formed_posts/${id}`, config )
-		},
 
-		importLibPost( data = {}, config = {} ) {
+		overrideLibPost(id, itemId, overrideType, config = {} ) {
```

Comment:
Let's change the method name here to `syncFromLibrary`.

Reply:
Altered name